packages/daemon/templates/wasm/predict.js

```diff
@@ -0,0 +1,41 @@
+const tvmjs = require("./tvmjs.bundle");
+const EmccWASI = require("./model.wasi");
+const fs = require('fs');
```

Comment:
This seems not working on the browser?

Reply:
IMO, a separate script is needed for browser env as we need to have file I/O in either case. And this part is not compatible.

Reply:
IMO it should be compatible because the developer is not expected to type the target when to start a pipeline.

Reply:
True, I will try to figure out a way to make the I/O part compatible. The fallback solution is to generate two entry files: one for node, another one is for browser